README.md

```diff
@@ -184,6 +184,44 @@ For example:
 
 The SAS Viya platform customizations that are managed by viya4-deployment are located under the [templates](https://github.com/sassoftware/viya4-deployment/tree/main/roles/vdm/templates) directory. These are purposely templatized and included there since they contain a set of customizations that are common or required for a functioning SAS Viya platform deployment. These particular files are configured via exposed variables that are documented within [CONFIG-VARS.md](docs/CONFIG-VARS.md) and do not need to be manually placed under `/site-config`.
 
+#### Base kustomization.yaml ConfigMap and Secret Generators
+
+In some scenarios, documentation instructs you to add a `configMapGenerator` or `secretGenerator` entry to the base `kustomization.yaml` (also known as `$deploy/kustomization.yaml`). For example:
```

Comment:
Could adding "In some scenarios, SAS Viya feature documentation instructs you to"
help clarify where you might get such an instruction?

It could also help tie the "with the documented" on line 213 back to an origin.

Reply:
After discussion on the phrasing with doc, I'll adjust this to:

> In some scenarios, a README or the deployment documentation instructs you...